### ISSUE TITLE
find-debuginfo reports no build ID note found error during rpmbuild

### ISSUE THREAD

**Issue by wrrobin** [Sandia-OpenSHMEM/SOS#957]
rpmbuild fails with the following error (not reproducible in Travis)
```
+ /usr/lib/rpm/find-debuginfo.sh --strict-build-id -m --run-dwz --dwz-low-mem-die-limit 10000000 --dwz-max-die-limit 110000000 /home/rahmanmd/rpmbuild/BUILD/sandia-openshmem-1.4.5
extracting debug info from /home/rahmanmd/rpmbuild/BUILDROOT/sandia-openshmem-1.4.5-1.el7.x86_64/usr/shmem/lib64/libpmi_simple.so.0.0.0
extracting debug info from /home/rahmanmd/rpmbuild/BUILDROOT/sandia-openshmem-1.4.5-1.el7.x86_64/usr/shmem/lib64/libsma.so.0.0.0
*** ERROR: No build ID note found in /home/rahmanmd/rpmbuild/BUILDROOT/sandia-openshmem-1.4.5-1.el7.x86_64/usr/shmem/lib64/libsma.so.0.0.0
error: Bad exit status from /var/tmp/rpm-tmp.CwP5x4 (%install)
```
A possible workaround is to stop building the debuginfo package by adding the following in the .spec file:
```
%define debug_package %{nil}
```
This work-around is validated to work and does not cause failure as above.

**Comment by wrrobin** [comment]
@jdinan Do you have any positive or negative opinion on the workaround? It seems to be ok to build without debuginfo for the RPM, but not sure what would be the implications, if any?

**Comment by jdinan** [comment]
The spec file is there to support OPA fabric software folks. You could check with them if they are ok with this solution.

**Comment by wrrobin** [comment]
Resolved by #962
